src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -76,6 +80,11 @@ export const ReserveConfiguration: React.FC<ReserveConfigurationProps> = ({ rese
             </Warning>
           )
         )}
+        {reserve.symbol === 'stETH' && (
```

Comment:
Same here

Reply:
I think it should be here since this page can work standalone (you can enter directly and supply directly from the reserve page)  and you should know that you're limited to certain actions.

Reply:
The reserve page will launch the supply modal which can have a warning still